src/AveragesList.js

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+
+const AveragesList = styled.div`
+color: #263238;
+> div {
```

Comment:
I think this was carried over from the other branch and I don't think it's valid CSS.